packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.8.4+6
+
+* Ensures the `result` callback and method call `arguments` send to iOS are 
+taken into consideration when picking multiple images on pre-iOS 14 devices;
+* Configures the `UIImagePicker` to default to gallery instead of camera on 
+pre-iOS 14 devices when picking multiple images on pre-iOS 14 devices.
```

Comment:
Nit: you say "on pre-iOS 14 devices" twice in this sentence.

Reply:
Removed the redundant "on pre-iOS 14 devices".